tests/e2e/specs/tree.cy.js

```diff
@@ -218,7 +218,7 @@ describe('Tree view', () => {
           .contains(name)
```

Comment:
Im not sure if this really needs additional tests. All the old tests for GScan - tree and table are passing